mysql-test/t/bug_ps7923.test

```diff
@@ -0,0 +1,30 @@
+--connect (con1,localhost,root,,)
```

Comment:
It would be good to add:
```
--source include/count_sessions.inc
--source include/wait_until_count_sessions.inc
```

Reply:
Not needed as we are restarting the server with `--source include/kill_and_restart_mysqld.inc`